.travis.yml

```diff
@@ -0,0 +1,55 @@
+# https://docs.haskellstack.org/en/stable/travis_ci/
+
+dist: bionic
+language: generic
+cache:
+  directories:
+  - $HOME/.stack
+  - $HOME/codeworld-account/.stack-work
+  - $HOME/codeworld-api/.stack-work
+  - $HOME/codeworld-auth/.stack-work
+  - $HOME/codeworld-compiler/.stack-work
+  - $HOME/codeworld-error-sanitizer/.stack-work
+  - $HOME/codeworld-game-api/.stack-work
+  - $HOME/codeworld-game-server/.stack-work
+  - $HOME/codeworld-prediction/.stack-work
+  - $HOME/codeworld-server/.stack-work
+
+addons:
+  apt:
+    packages:
+      - libgmp-dev
+
+matrix:
+  include:
+  - name: "root (no tests)"
+    env: MODULE=. TEST=false
+  - name: "account"
+    env: MODULE=codeworld-account TEST=true
+  - name: "api"
+    env: MODULE=codeworld-api TEST=true
+  - name: "auth"
+    env: MODULE=codeworld-auth TEST=true
+  - name: "compiler (no tests)"
+    env: MODULE=codeworld-compiler TEST=false
```

Comment:
This is particularly unfortunate, since these tend to be the tests that I care the most about.  But it's not unexpected, since this would require the patched ghcjs built by the install script.